materialize-sql-json/.snapshots/TestTableTemplate

```diff
@@ -0,0 +1,18 @@
+
+	CREATE TABLE one."reserved".checkpoints (
+			materialization TEXT NOT NULL,
```

Comment:
This snapshot change (the part with the `NOT NULL` added) is not a result of this PR. It's from running re-running the sqlgen tests, which apparently don't ever get run in CI.